### ISSUE TITLE
[Accessibility] Mouse echo don't work with NVDA and  JAWS

### ISSUE THREAD

**Issue by melisaAkel** [melisaAkel/vscode#315]
Original issue: https://github.com/microsoft/vscode/issues/247522

Does this issue occur when all extensions are disabled?: Yes/No
yes

- VS Code Version: >1.92
- OS Version: current Windows 11 

Steps to Reproduce:

1. Open Visual Studio Code version 1.92 (or later).
2. Ensure that a Screen Reader is running (e.g., NVDA, JAWS, or ZoomText).
3. Make sure that the "Mouse Echo" feature is enabled in the Screen Reader settings (it reads text under the mouse pointer).
4. Move the mouse pointer over text in the editor area of VSCode.

5. Observe that the Screen Reader does not announce or read the text under the mouse, even though it should.

In Version June 2024 or fewer it works.
